homeassistant/components/samsungtv/device_trigger.py

```diff
@@ -38,7 +42,7 @@ async def async_validate_trigger_config(
     """Validate config."""
     config = TRIGGER_SCHEMA(config)
 
-    if config[CONF_TYPE] == TURN_ON_PLATFORM_TYPE:
+    if [TURN_ON_PLATFORM_TYPE, TURN_OFF_PLATFORM_TYPE].count(config[CONF_TYPE]) == 1:
```

Comment:
`config[CONF_TYPE] in [TURN_ON_PLATFORM_TYPE, TURN_OFF_PLATFORM_TYPE]`
is better and readable way of saying 'it should be one of TURN_ON, TURN_OFF types'.

Reply:
Thank you. That's much more readable indeed.